src/Symfony/Component/Yaml/Exception/ParseException.php

```diff
@@ -103,7 +103,7 @@ public function getParsedLine()
      *
      * @param int $parsedLine The file line
```

Comment:
Don't know what's the policy ... but shouldn't we remove PHPdoc comments like this that now don't provide any meaningful value? Thanks.

```
@param int $parsedLine The file line
```

Reply:
Yes, the policy is to remove those annotations.